src/Makefile.am

```diff
@@ -669,6 +669,16 @@ mbmon_la_LIBADD += -lsocket
 endif
 endif
 
+if BUILD_PLUGIN_MCELOG
+pkglib_LTLIBRARIES += mcelog.la
+mcelog_la_SOURCES = mcelog.c
+mcelog_la_LDFLAGS = $(PLUGIN_LDFLAGS)
+mcelog_la_LIBADD =
+if BUILD_WITH_LIBSOCKET
```

Comment:
I don't think you need this, it is mainly for Solaris iirc, and mcelog seems Linux-only.